actions/templateRegistry.js

```diff
@@ -13,39 +13,39 @@ const axios = require('axios').default;
 const { Octokit } = require('@octokit/rest');
 const { v4: uuidv4 } = require('uuid');
 
+const collectionName = 'templates';
+
 const TEMPLATE_STATUS_IN_VERIFICATION = 'InVerification';
 const TEMPLATE_STATUS_APPROVED = 'Approved';
 const TEMPLATE_STATUS_REJECTED = 'Rejected';
 
 let openReviewIssues = null;
 
+const { mongoConnection } = require('../db/mongo');
+
 /**
  * Returns a template record from Template Registry by a template name.
  *
+ * @param {object} dbParams database connection parameters
  * @param {string} templateName template name
- * @param {string} templateRegistryOrg Template Registry organization name
- * @param {string} templateRegistryRepository Template Registry repository name
  * @returns {Promise<object|null>} an existing template record or null
  */
-async function findTemplateByName(templateName, templateRegistryOrg, templateRegistryRepository) {
-  const templates = await getTemplates(templateRegistryOrg, templateRegistryRepository);
-  const template = templates.find(item => item.name === templateName);
-  return (template !== undefined) ? template : null;
+async function findTemplateByName(dbParams, templateName) {
+  const collection = await mongoConnection(dbParams, collectionName);
+  const results = await collection.find({ 'name': templateName }).toArray();
+  return results.length ? results[0] : null;
 }
 
 /**
  * Adds a template to Template Registry.
  *
+ * @param {object} dbParams database connection parameters
  * @param {string} templateName template name
  * @param {string} githubRepoUrl Github repo URL
- * @param {string} githubAccessToken Github access token
- * @param {string} templateRegistryOrg Template Registry organization name
- * @param {string} templateRegistryRepository Template Registry repository name
- * @param {string} commitMessage commit message
  * @returns {object} a newly created template
  */
-async function addTemplate(templateName, githubRepoUrl, githubAccessToken, templateRegistryOrg, templateRegistryRepository, commitMessage) {
-  const templates = await getTemplates(templateRegistryOrg, templateRegistryRepository, true);
+async function addTemplate(dbParams, templateName, githubRepoUrl) {
+  const collection = await mongoConnection(dbParams, collectionName);
   const template = {
     'id': uuidv4(),
```

Comment:
Can we use built in `_id` instead of this? 

Note: We may need to convert the MongoDB id to a string before returning, see: https://stackoverflow.com/a/24558372/22265578